i18n/zh-CN/docusaurus-plugin-content-docs/current/crab-parameters.md

```diff
@@ -76,11 +71,11 @@ sidebar_label: Crab网络参数
 | Proposal Bond               | 5% and minumum 1000 CRING | The amount required to bond in order to propose a treasury spend. If approved, it is returned, if the proposal fails, it is burnt. |
 | Burn unspent treasury funds | Off                       | This deactivates a burn of all  unspent treasury funds at the end of a budgeting period.                                           |
```

Comment:
@AurevoirXavier Did you have some time to help me translate those table params into Chinese?

Reply:
Would like to open another PR

Reply:
Do it!